src/core/control/Control.h

```diff
@@ -75,7 +75,7 @@ class Control:
         public ClipboardListener,
         public ProgressListener {
 public:
-    Control(GApplication* gtkApp, GladeSearchpath* gladeSearchPath);
+    Control(GApplication* gtkApp, GladeSearchpath* gladeSearchPath, gboolean disableAudio);
```

Comment:
```suggestion
    Control(GApplication* gtkApp, GladeSearchpath* gladeSearchPath, bool disableAudio);
```
convertion from gboolean to bool is safe (and is done anyway implicitely in your `? :` construct)

Reply:
done